Web/Edubase.Web.UI/Views/Shared/Downloads/DownloadError.cshtml

```diff
@@ -36,7 +36,7 @@
 {
     @if (Model.SearchQueryString.IsNullOrWhiteSpace() || Model.SearchSource == null)
     {
-        @Html.ActionLink("Search", "Index", "Search", new { area = "" }, new { @class = "govuk-back-link" })
+        @Html.ActionLink("Search", "Index", "Search", new { area = "" }, new { @class = "govuk-breadcrumbs__link" })
```

Comment:
In this scenario I think we'll want to include the divs (similar to the other links within the ELSE statement).

An easy way to test what this renders like, is to go to the main downloads page, select a small filesize file, and then click through until you download it. If you click on the Results.zip button a second time it will show this error page with the empty Search link at the top.